threshold.js

```diff
@@ -15,6 +15,7 @@ import * as jsQUEST from "./lib/jsQUEST.module.js";
 window.jsQUEST = jsQUEST;
 
 var conditionTrials;
+var blocksPerTrial;
```

Comment:
What do you mean by blocks per trial? A block contains multiple trials.

Reply:
Okay, I thought it was the otherway. Then it should have been trialPerBlocks?